mco/machine-config-server.deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
```

Comment:
This deployment and it's associated service and route are optional and should only be used for testing.  Creating this route exposes the ignition payload to anything that has access to the public API, which is probably undesireable in most cases.